src/css/tailwind.scss

```diff
@@ -5,10 +5,11 @@
 .theme-doc-markdown a, .fancy-link {
     @apply no-underline bg-gradient-to-r from-primary-400 to-primary-400 bg-no-repeat bg-left-bottom 
 		bg-0w hover:bg-100w hover:text-primary-500 duration-300 w-fit text-primary-200 transition-all;
-  }
+}
+
 .theme-doc-markdown {
 
-  *[aria-hidden="true"] {
+  *[aria-hidden="false"] {
```

Comment:
this should not be changed, maybe a debug statement?

if aria hidden is set to true, hidden shall be applied.

Reply:
Yes remove the whole block.